style.css

```diff
@@ -0,0 +1,71 @@
+html
+{
+    font-size: 10px;
+}
+body
+{
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto',
+      'Helvetica Neue', Arial, sans-serif;
+    font-size: 1.6rem;
+    line-height: 1.5;
+    text-align: center;
+    color: #333;
+    margin: 0;
+}
+
+h1
+{
+    font-size: 4rem;
+    margin-bottom: 0;
+}
+
+@media (max-width:460px)
+{
+    h1
+    {
+        font-size: 3.6rem;
+      line-height: 1.2;
+    }
+}
+
+h2 
+{
+    font-size: 3.25rem;
+}
+
+#main 
```

Comment:
please avoid using ids as css selectors to avoid repeating the same code if anaother element uses the same style.

Reply:
Nice job explaining the reason for not using ids as css selectors